.github/workflows/build-vineyardd-and-wheels-linux.yaml

```diff
@@ -268,19 +262,6 @@ jobs:
           aarch64-linux-gnu-strip ./vineyardctl || true
           upx -9 ./vineyardctl || true
 
-      - name: Upload vineyardctl to tagged release
```

Comment:
Check this as well.

Reply:
I've reverted the previous hack commit, PTAL again.